publisher/utils/visualize_publishers.py

```diff
@@ -14,7 +14,7 @@ async def main(pair):
     decimals = await client.get_decimals(key)
     entries = []
 
-    num_reps = 12 * 3  # Run for 3 hours
+    num_reps = 1  # 2 * 3  # Run for 3 hours
     for i in range(num_reps):
         print("awake - fetching")
         entries.extend(await client.get_entries(key))
```

Comment:
need to pass in empty list of sources

Reply:
No because SDK handles sources=None gracefully